client/me/help/help-courses/course.jsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import Card from 'components/card';
+
+export class Course extends Component {
+	render() {
+		return (
+			<Card className="help-courses__course">
+				Latest course: Only visible for users with the business plan
+			</Card>
+		);
+	}
+}
+
+export const CoursePlaceholder = () => {
+	return (
+		<Card className="help-courses__course is-placeholder"></Card>
```

Comment:
I remember way back hearing that these should be kept separate. I cant remember who or when but it happened.